cli/tests/unit/command_test.ts

```diff
@@ -0,0 +1,644 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import {
+  assert,
+  assertEquals,
+  assertRejects,
+  assertStringIncludes,
+  assertThrows,
+} from "./test_util.ts";
+
```

Comment:
Add a test for `Deno.spawn`/`Deno.spawnSync` that override `options.stdout` and `options.stderr` to showcase that they will always be set to "piped" internally